src/mailing/sender.js

```diff
@@ -0,0 +1,35 @@
+const nodemailer = require('nodemailer');
+require('dotenv').config();
```

Comment:
Do we need this usually or is it just for testing? I would expect that this would usually already have been loaded when the app starts up by `expressServer.js`

Reply:
Removed